packages/addon/lua/gelly/api/mods/enums.lua

```diff
@@ -4,6 +4,6 @@ gellyx.mods = gellyx.mods or {}
 ---@enum gellyx.mods.ModType
 --- Enum for mod types.
 gellyx.mods.ModType = {
-	--- A mod that acts in a global context. (e.g: a mod that rains liquid or a lava game mode.)
+	--- A mod that acts in a global context. (e.g: a mod that rains liquid or a lava game mode)
 	Global = 1,
```

Comment:
.. I don't think this should be pushed with *just* a global mod type. What about something like a fluid grabber, which works independently of the currently loaded fluid preset?